### PR TITLE
initial files for LLD

### PR DESCRIPTION
**Pull request by pmandler**
Apparently I forgot to check these in weeks ago.

### CODE REVIEW

File: venues/ICLR.cc/2019/Workshop/LLD/python/init.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+print('connecting to {0}'.format(client.baseurl))
+
+conference = config.get_conference(client)
+conference.open_submissions(due_date = datetime.datetime(2019, 3, 16, 0, 0))
+
+'''
+Create the homepage and add it to the conference group.
+'''
+conf_group = client.get_group(conference.get_id())
+conf_group.add_webfield('../webfield/conferenceWebfield.js')
```
Comment (melisabok) [inline venues/ICLR.cc/2019/Workshop/LLD/python/init.py]: Why do you need to overwrite the webfield?

File: venues/ICLR.cc/2019/Workshop/LLD/python/init.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+conf_group = client.post_group(conf_group)
+print("added webfield to", conf_group.id)
+
+
+'''
+Update process function on submission for blind submission
+'''
+submission_inv =client.get_invitation(conference.get_submission_id())
+with open('../process/submissionProcess.js') as f:
```
Comment (melisabok) [inline venues/ICLR.cc/2019/Workshop/LLD/python/init.py]: same here

File: venues/ICLR.cc/2019/Workshop/LLD/python/init.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    submission_inv.process = f.read()
+
+submission_inv.reply['content']['authorids']['description']+=" Please provide real emails; identities will be anonymized."
+submission_inv.reply['content']['authors']['description']+=" Please provide real names; identities will be anonymized."
+submission_inv = client.post_invitation(submission_inv)
+print("posted invitation "+submission_inv.id)
+
+blind_submission_inv = invitations.Submission(
```
Comment (melisabok) [inline venues/ICLR.cc/2019/Workshop/LLD/python/init.py]: we can use the builder to create the blind submissions

**Comment by pmandler** [comment]
These files were made in January before blind submissions were handled by the builder.  I can rewrite this to be current, or we can just merge it in the way that it was used at the time.

**Comment by melisabok** [comment]
ok, but I don't see the web field and process functions in the repo.

**Comment by pmandler** [comment]
Good point.  I've added them.

**Pull request merged**